### PR TITLE
Added Favourite Page, Test

### PR DESCRIPTION
**Pull request by reachtonavakanth**
Heyy,

Added Favourite Pages & Tests, Please review and action on this request.

### CODE REVIEW

**Comment by Navakanthtunga** [comment]
Will pick in 2nd Half

File: src/test/java/com/webcrawler/pages/FavouritePage.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.webcrawler.pages;
+
+public class FavouritePage {
```
Comment (Navakanthtunga) [inline src/test/java/com/webcrawler/pages/FavouritePage.java:3]: Hey pls add Remove Favourite as per recent call with product

File: src/test/java/com/webcrawler/tests/FavouriteTest.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package com.webcrawler.tests;
+
+public class FavouriteTest {
```
Comment (Navakanthtunga) [inline src/test/java/com/webcrawler/tests/FavouriteTest.java:3]: Hey pls add Remove Favourite as per recent call with product

File: src/test/java/com/webcrawler/pages/FavouritePage.java [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package com.webcrawler.pages;
```
Comment (Navakanthtunga) [inline src/test/java/com/webcrawler/pages/FavouritePage.java:1]: Check Packages

**Review by Navakanthtunga — COMMENTED** [review]
Code looks good, Please do modifications as per request

**Review by Navakanthtunga — CHANGES_REQUESTED** [review]
Hey Please incorporate chnages by EOD

File: src/test/java/com/webcrawler/pages/FavouritePage.java (hunk 1 above)
Comment (reachtonavakanth) [reply src/test/java/com/webcrawler/pages/FavouritePage.java:3]: Added.

File: src/test/java/com/webcrawler/tests/FavouriteTest.java (hunk 2 above)
Comment (reachtonavakanth) [reply src/test/java/com/webcrawler/tests/FavouriteTest.java:3]: Added

**Review by Navakanthtunga — APPROVED** [review]
Approved, Pls merge to master.

**Comment by reachtonavakanth** [comment]
Added Favourite Page, Test

**Pull request merged**